.pipelines/templates/publish-helm-chart.yaml

```diff
@@ -5,11 +5,25 @@ steps:
     displayName: Login to acr
     env: 
       REGISTRY_NAME: $(REGISTRY_NAME)
+  - script: |
+      cp $(image.dir)/$(chart.name)-*.tgz .
```

Comment:
Can you see about just using the helm chart in the original directory instead of copying?